ha/core/controllers/pcs/node_controller.py

```diff
@@ -50,16 +50,65 @@ def initialize(self, controllers):
+                Log.debug(f"Node {node_name} is already online")
+                return {"status": const.STATUSES.SUCCEEDED.value, "output": NODE_STATUSES.ONLINE.value, "error": ""}
+            elif node_status == NODE_STATUSES.STANDBY.value or node_status == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value:
+                # Unstandby the node
+                if self.heal_resource(node_name):
+                    _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_UNSTANDBY.replace("<node>", node_name), check_error=False)
+                    if _rc != 0:
+                        Log.error(f"Failed to start node {node_name}, Error: {_err}")
+                        return {"status": const.STATUSES.FAILED.value, "output": "", "error": f"Failed to start node {node_id}, Error: {_err}"}
+                    Log.debug(f"Node {node_name} was in standby mode, unstandby operation started successfully")
+                else:
+                    Log.error(f"Node {node_name} is in standby mode : Resource failcount found on the node, cleanup did not work")
+                    return {"status": const.STATUSES.FAILED.value, "output": "", "error": f"Node {node_id} is in standby mode, resource failcount found on the node, cleanup did not work"}
+            elif node_status == NODE_STATUSES.CLUSTER_OFFLINE.value:
+                _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_START.replace("<node>", node_name), check_error=False)
+                if _rc != 0:
+                    Log.error(f"Failed to start node {node_name}, Error: {_err}")
+                    return {"status": const.STATUSES.FAILED.value, "output": "", "error": f"Failed to start node {node_id}, Error: {_err}"}
+                Log.debug(f"Node {node_name} started successfully. Waiting for cluster to stabalize and then get the node status")
+                time.sleep(const.BASE_WAIT_TIME * 2)
+                # Get the status of the node again
+                node_status = self.nodes_status([node_name])[node_name]
+                # If the node is in standby mode, unstandby here
+                if node_status == NODE_STATUSES.STANDBY.value:
+                    Log.warn(f'Node {node_name} is still in standby mode')
+                    _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_UNSTANDBY.replace("<node>", node_name), check_error=False)
+                    if _rc != 0:
+                        Log.error(f"Failed to start node {node_name}, Error: {_err}")
+                        return {"status": const.STATUSES.FAILED.value, "output": "", "error": f"Failed to start node {node_id}, Error: {_err}"}
+            else:
+                Log.error(f"{node_name} status is {node_status}, node cannot be started.")
+                return {"status": const.STATUSES.FAILED.value, "output": "", "error": f"Node {node_id} status is {node_status}, node cannot be started."}
+
+            # Update the node status in system health
```

Comment:
The jira mentions that enclosure health should also be updated .
>Node and enclosure health should be updated in the system health.

Reply:
Added TODO, we are not updating the enclosure health in stop as well.